### PR TITLE
SGSSB Entries

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

**Comment by DeathlyPlays** [comment]
WIP Do not merge

**Comment by ghost** [comment]
Ok someone needs to fix these Travis errors >_>

**Comment by DeathlyPlays** [comment]
These errors aren't caused by me @WonderGummi

**Comment by ghost** [comment]
My point still stands

**Comment by DeathlyPlays** [comment]
My point says there's no Travis issue with my code @WonderGummi and I refractored it feelsmd

File: spacialgaze-plugins/sgssb.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	sgssb: function (target, room, user) {
+		if (!this.runBroadcast()) return false;
+		if (!target || target === 'help') return this.parse('/help sgssb');
+		if (target === 'credits') return this.parse('/sgssbcredits');
+		let targetData = getMonData(toId(target));
+		if (!targetData) return this.errorReply("The staffmon '" + toId(target) + "' could not be found.");
+		return this.sendReplyBox(targetData);
+	},
+
+	sgssbhelp: function (target, room, user) {
```
Comment (HoeenCoder) [inline spacialgaze-plugins/sgssb.js]: this can be setup as a help command instead of a whole new command.

File: spacialgaze-plugins/sgssb.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		if (!this.runBroadcast()) return;
+		return this.sendReplyBox("/sgssb [staff member\'s name] - displays data for a staffmon\'s movepool, custom move, and custom ability.");
+	},
+
+	sgssbcredits: function (target, room, user) {
```
Comment (HoeenCoder) [inline spacialgaze-plugins/sgssb.js]: We don't need to add a credits popup for this.

**Comment by ghost** [comment]
Well its fixed now.

here is mine

VXN
(Rayquaza)
@ Normalium Z
Ability: Wonder Guard
Jolly Nature
EVs: 252 Atk / 252 Spe / 4 SpD
Moves: Splash, Happy Hour, Celebrate
Signature Move: Crash
Signature Move Effect: Displays a server crash message and faints the user.

**Review by HoeenCoder — DISMISSED** [review]
I don't think a whole file for this is nessecary (the command i mean), since ssb.js handles SSBFFA, add this to misc.js instead.

**Comment by ghost** [comment]
@DeathlyPlays  can I have collaborator on your sg repo to help

File: spacialgaze-plugins/misc.js [hunk 3]
```diff
@@ -7,6 +7,29 @@
  * @license MIT license
  */
 'use strict';
+'use strict';
```
Comment (ghost) [inline spacialgaze-plugins/misc.js]: Ok, there is no way in the entirety of the world where you should have 2 `'use strict';`

Comment (DeathlyPlays) [reply spacialgaze-plugins/misc.js]: oops lmao

**Comment by DeathlyPlays** [comment]
@HoeenCoder look good so far?

**Comment by HoeenCoder** [comment]
Yeah be sure to test it ofc

**Pull request merged**